### PR TITLE
feat(handler): Trevorbot can now tell Chuck Norris facts

### PR DESCRIPTION
**Pull request by evansiroky**
- Trevorbot will fetch a random Chuck Norris fact from
  http://www.icndb.com/ when asked.
- Added request timeout handlers

### CODE REVIEW

File: handler.js [hunk 1]
```diff
@@ -1,26 +1,66 @@
...
+        var country = sanitizeChars(data.location.now.country)
+        cb(null, { text: `Trevor is in ${city}, ${country} (https://nomadlist.com/trevorgerhardt)` })
       })
     } else {
       cb(null, { text: 'I only know where Trevor is.' })
     }
+  } else if (text.indexOf('chuck norris') > -1) {
+    makeRequest(http, 'api.icndb.com', '/jokes/random', (err, data) => {
+      if (err) { return cb(null, { text: 'I don\'t feel like doing that right now :pensive:' }) }
+      cb(null, { text: data.value.joke })
```
Comment (mattwigway) [inline handler.js]: Can we use sanitizeChars here?

**Review by mattwigway — CHANGES_REQUESTED** [review]

File: handler.js (hunk 1 above)
Comment (evansiroky) [reply handler.js]: What is the intent of removing characters?  The regex will remove a lot of stuff, such as numbers, punctuation, etc.

Comment (mattwigway) [reply handler.js]: You could expand the regex to include numbers and punctuation. It's to avoid having slack formatting passed through.

Comment (evansiroky) [reply handler.js]: ok, see update

**Review by mattwigway — APPROVED** [review]

**Pull request merged**